README.md

```diff
@@ -54,6 +54,16 @@ realtime.push.admin.publish(recipient, data: data) { error in
 
 Demos available: [macOS](https://github.com/ably/demo-macos) and [tvOS](https://github.com/ably/demo-tvos).
 
+## Known Limitations
+
+This client library is currently *not compatible* with some of the Ably features:
+
+| Feature |
+| :--- |
+| [Custom transportParams](https://ably.io/documentation/realtime/usage#client-options) |
```

Comment:
@Srushtika please stop using `ably.io` in domains.
Our domain is `www.ably.io`.  In presentations if you want to drop the www that is OK, but not in actual URLs.

```
$ curl -i https://ably.io/documentation/realtime/usage
HTTP/2 301
date: Thu, 14 Feb 2019 17:12:02 GMT
cache-control: max-age=3600
expires: Thu, 14 Feb 2019 18:12:02 GMT
location: https://www.ably.io/documentation/realtime/usage
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
server: cloudflare
cf-ray: 4a9128497fa7352a-LHR
```